columnflow/tasks/framework/base.py

```diff
@@ -448,6 +448,35 @@ def get_array_function_kwargs(cls, task=None, **params):
 
         return kwargs
 
+    @classmethod
+    def resolve_config_parameter(
+        cls,
+        config_inst: od.Config,
+        param: tuple[str] | None = None,
+        param_name: str | None = None,
+    ) -> tuple[str]:
+        """
+        Resolves tuple of inputs *param* by using groups defined in the *config_inst*
+        that map a string to a tuple of strings. If *param* is empty, the default
+        defined in the config is returned.
```

Comment:
In the doc string, could you mention

- the `f"default_{param_name}"` name of the default entry, and
- the `f"{param_name}_groups"` name of the group entry?

Reply:
Good point, I decided to instead implement these strings as input parameters `default_str` and `groups_str`